CHANGELOG.md

```diff
@@ -1,7 +1,23 @@
 # Changelog
 
+## [0.3.0] - 2023-02-20
 
-## [0.2.0] - 2023-01-09
+### Features
+- verifier sync data from eigenda, verify and generate new blocks([#638](https://github.com/mantlenetworkio/mantle/pull/638))
+- Add rollup to eigenda's data checker mechanism([#670](https://github.com/mantlenetworkio/mantle/pull/670))
```

Comment:
Make a verification mechanism for the data stored in da, and rollup again if the storage fails